vtysh/vtysh.h

```diff
@@ -121,4 +137,17 @@ extern int user_mode;
 
 extern bool vtysh_add_timestamp;
 
+struct vtysh_client {
+	int fd;
+	const char *name;
+	int flag;
+	char path[MAXPATHLEN];
+	struct vtysh_client *next;
+
+	struct thread *log_reader;
+	int log_fd;
+};
+
+extern struct vtysh_client vtysh_client[21];
```

Comment:
Isn't there a way we can avoid hard coding to `21`?

Reply:
See previous discussion. If you have suggestions I'm ears though!